opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -53,57 +47,56 @@ def collect(self) -> Optional[_PointVarT]:
         pass
 
 
-class SynchronousSumAggregation(
-    _InstrumentMonotonicityAwareAggregation, Aggregation[Sum]
-):
-    def __init__(self, instrument_is_monotonic: bool):
-        super().__init__(instrument_is_monotonic)
-        self._value = 0
+class SumAggregation(Aggregation[Sum]):
+    def __init__(
+        self,
+        instrument_is_monotonic: bool,
+        instrument_temporality: AggregationTemporality,
+    ):
+        super().__init__()
+        self._instrument_is_monotonic = instrument_is_monotonic
+
+        if instrument_temporality is AggregationTemporality.DELTA:
+            self._value = 0
+
+        else:
+            self._value = None
```

Comment:
What is the purpose of `self_value` being `None` originally? Was this to recognize that `aggregate` was never called before `collect`?

Reply:
Fixed